twisted/conch/scripts/ckeygen.py

```diff
@@ -33,10 +33,10 @@ class GeneralOptions(usage.Options):
                      ['type', 't', None, 'Specify type of key to create.'],
                      ['comment', 'C', None, 'Provide new comment.'],
                      ['newpass', 'N', None, 'Provide new passphrase.'],
-                     ['pass', 'P', None, 'Provide old passphrase.']]
+                     ['pass', 'P', None, 'Provide old passphrase.'],
+                     ['format', 'l', 'md5-hex', 'Fingerprint format of key file.']]
```

Comment:
what was wrong with `fingerprint` as the name of this parameter?

btw, this is already changing a UX, so I think that it safe to use sha2 based format